lua/wikis/commons/Icon/Data.lua

```diff
@@ -205,6 +205,11 @@ return {
 	find_images = 'fad fa-file-search',
 	clipboard = 'fad fa-clipboard-list',
 
+	-- Usage: Main Pages navcards
+	players = 'fas fa-user',
```

Comment:
Players and being single user doesn't make sence to me, either both singular or both plural